### DISCUSSION TITLE
Hide support message at the top of the window?

### DISCUSSION THREAD

**Discussion by jtssi** [sandboxie-plus/Sandboxie#2788] [General]
![2023-04-05_10 55 37](https://user-images.githubusercontent.com/71876429/230033054-516a15b7-edfa-444c-b0ac-058544bd535c.jpg)

Hello, when you buy a license, is the green message at the top hidden?
Thank you very much.

**Comment by hg421** [comment]
Yes. This link is hidden after you add a support certificate.

**Reply by jtssi** [reply]
support certificate and/or license?

**Reply by hg421** [reply]
> support certificate and/or license?

It is basically the same thing, if you buy a license from xanasoft.com (or subscribe on Patreon, or contribute to the project otherwise) you get a "support certificate" that you can add in Global Options -> Support
